forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -119,6 +121,14 @@ else if (c.hasKeyword(Keyword.WITHER)) {
             value += addValue(c.getKeywordMagnitude(Keyword.TOXIC) * 5, "toxic");
             value += addValue(c.getKeywordMagnitude(Keyword.AFFLICT) * 5, "afflict");
             value += addValue(c.getKeywordMagnitude(Keyword.RAMPAGE), "rampage");
+
+            if (c.hasTriggerThatMatches(Arrays.asList(
+                    TriggerType.DamageDealtOnce,
+                    TriggerType.DamageDone,
+                    TriggerType.DamageDoneOnce,
+                    TriggerType.DamageDoneOnceByController))) {
+                value += addValue(10, "cdamage_trigger");
```

Comment:
imo this is still way too inaccurate

what about cards like _Filthy Cur_? arguably they might be fewer but this would still introduce other misplays

also we're already tagging cards like this with SVar:Has[Attack/Combat]Effect

you could try to study their triggers to find the common parts for matching, so that manual labor is no longer required

Reply:
It's still better than it was. I see no easy way to implement that. I tried just implementing it for a generic curiosity effect, but even trying to match that specifically by just looking at the triggers was extremely complicated.